simple_3d_game.py

```
@@ -0,0 +1,45 @@
+import pyglet
+from pyglet.gl import *
```

Ruff (0.11.9)

`from pyglet.gl import *` used; unable to detect undefined names

(F403)